src/components/EntityPage/Participant/ParticipantClinical.js

```diff
@@ -154,7 +155,7 @@ class ParticipantClinical extends React.Component {
   }
 
   showParticipantNb = (wrapper, field, value) => {
-    if (wrapper.value === '0 participants') return <div>0 participant</div>;
+    if (wrapper.value === '0 participants') return <div>0 participants</div>;
```

Comment:
do you really need a specific rule for 0 participant?

Reply:
Actually, yes we need thius. It replace the link by a div

Reply:
as the code stands right now, I would say yes.